test/cpp/util/cli_credentials.cc

```diff
@@ -49,7 +54,31 @@ std::shared_ptr<grpc::ChannelCredentials> CliCredentials::GetCredentials()
     if (FLAGS_use_auth) {
       return grpc::GoogleDefaultCredentials();
     } else {
-      return grpc::SslCredentials(grpc::SslCredentialsOptions());
+      auto options = grpc::SslCredentialsOptions();
+      if (!FLAGS_ssl_server_ca_file.empty()) {
+        std::stringstream ca_buffer;
+        ca_buffer << std::ifstream(FLAGS_ssl_server_ca_file).rdbuf();
+
+        if (ca_buffer.str().empty()) {
+          exit(2);
+        }
+        options.pem_root_certs = ca_buffer.str();
+      }
+      if (!FLAGS_ssl_client_cert_file.empty() && !FLAGS_ssl_client_key_file.empty()) {
```

Comment:
Please handle the case when was is empty and the other is not and warn the user that this is an invalid combination.